src/services/UpdateBalance/index.tsx

```diff
@@ -20,8 +20,7 @@ const fetcher = async (
 ): Promise<FilecoinNumber | null> => {
   if (!apiAddress || !actorAddress) return null
 
-  if (!validateAddressString(actorAddress))
-    throw new Error('Invalid actor address')
+  if (!isFilAddress(actorAddress)) throw new Error('Invalid actor address')
```

Comment:
idea: what if we used the eth json rpc if balance poller receives an eth address? might be useful for building eth apps?